pyproject.toml

```
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "backend-skeleton"
```

_(minor)_
```suggestion
name = "DjangoStars Backend Skeleton"
```
Or somethis like this 🤔